src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java

```diff
@@ -95,34 +95,41 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example of using escapes with trail comment:
+ * Example of using escapes for non-printable, control characters:
  * </p>
  * <pre>
- * String unitAbbrev = "&#92;u03bcs"; // Greek letter mu, "s"
- * String textBlockUnitAbbrev = """
- *          &#92;u03bcs"""; // Greek letter mu, "s"
+ * String unitAbbrev = "\ufeff"; // OK
  * </pre>
- * <p>An example of how to configure the check to allow using escapes
+ * <p>
+ * An example of how to configure the check to allow using escapes
  * if trail comment is present:
  * </p>
  * <pre>
  * &lt;module name="AvoidEscapedUnicodeCharacters"&gt;
  *   &lt;property name="allowByTailComment" value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
- * <p>Example of using escapes if literal contains only them:
+ * <p>Example of using escapes if trail comment is present:
  * </p>
  * <pre>
- * String unitAbbrev = "&#92;u03bc&#92;u03bc&#92;u03bc";
+ * String unitAbbrev = "\u03bcs"; // Greek letter mu, "s"
```

Comment:
please keep escaping `&#92` to woraround Teamcity validator.

Reply:
done